src/components/Impact/ImpactUnitDialog/ImpaceUnitDialog.tsx

```diff
@@ -242,7 +242,7 @@ function ImpactUnitDialog({
 		}
 	);
 
-	const initialValues = formAction == FORM_ACTIONS.CREATE ? defaultValues : formValues;
+	const initialValues = formAction === FORM_ACTIONS.CREATE ? defaultValues : formValues;
 	useEffect(() => {
 		if (impactCategories) {
```

Comment:
what do these lines do? 
Could be extracted to a function with explanation

Reply:
I am checking if we are creating or updating, I am assigning the values to initialValues so the person reading the code would know I am setting initialValues.